ENH: Citing source in README file

the-nose-knows opened the pull request:
For GH users who strictly or heavily use the web-view instead of a local Git, having a direct link is handy, as it does not require downloading the PDF _if_ the user wanted to go to the source of it directly. It's an alternative that allows those interested in more uploads similar to this PDF from the same author(s).

 - [ ] closes: N/A
 - [ ] tests added / passed: N/A
 - [ ] whatsnew: Added source of PDF to README file for the cheatsheet

the-nose-knows committed:
Citing source in README file

For GH users who strictly or heavily use the web-view instead of a local Git, having a direct link is handy, as it does not require downloading the PDF _if_ the user wanted to go to the source of it directly. It's an alternative that allows those interested in more uploads similar to this PDF from the same author(s).

jorisvandenbossche commented:
You linked to the R cheatsheet (on which the pandas one is based). Was that on purpose?

jreback commented:
actually I think this is reasonable, though I think the wording should be that this is *referenced/based* on the dplyr sheet.

the-nose-knows committed:
jorisvandenbossche's feedback

I re-read the PDF and made sure the wording reflected the content presented. I also changed the source-citing so that is more friendly for .TXT files instead of Markdown or unspecified.

the-nose-knows commented:
@jorisvandenbossche Good catch! I've updated the file with a new commit.
@jreback feel free to word this to whatever is most appropriate. As long as the spirit of the PR is satisfied, I'm happy with whatever implementation is used.

the-nose-knows committed:
Update README.txt

the-nose-knows committed:
English enhancement

Improved sentence structure for English speakers.

jreback commented:
seems reasonable, thanks.
